settings.gradle

```diff
@@ -10,6 +10,7 @@ project(':threadutils').projectDir = new File(rootDir, 'components/utils/threadu
 project(':cachedrequestloader').projectDir = new File(rootDir, 'components/utils/cachedrequestloader')
 project(':telemetry-annotation').projectDir = new File(rootDir, 'components/service/telemetry-annotation')
 project(':telemetry-compiler').projectDir = new File(rootDir, 'components/service/telemetry-compiler')
+project(':permissionhandler').projectDir = new File('components/utils/android-permission-handler/permissionhandler')
```

Comment:
If you want to make it an Android component in the future, you may want to change the path.